package.json

```
@@ -1,26 +1,27 @@
 {
-    "name": "alert-hub",
+    "name": "ifrc-alert-hub",
     "version": "0.0.0",
     "type": "module",
     "private": true,
     "scripts": {
         "start": "vite",
         "build": "vite build",
+        "preview": "view preview",
```

Might be wrong, but isn't it:
```suggestion
        "preview": "vite preview",
```